internal/circonus/metric_filters.go

```diff
@@ -85,10 +66,103 @@ func (c *Check) defaultFilters() [][]string {
     ["allow", "^events$", "events"],
     ["allow", "^collect_.*$", "agent collection stats"],
     ["allow", "^authentication_attempts$", "api auth health"],
+    ["allow", "^cadvisor.*$", "cadvisor"],
     ["deny", "^.+$", "all other metrics"]
     ]
 }
-`)
+`
+	defaultMetricFiltersStr118 = `
```

Comment:
Same as above: In my latest research I discovered that one metric change didn't happen until 1.20, so I'm pegging my new dashboard to a v1.20 minimum, so you should do so also, I think.

Reply:
resolved in [d71873a](https://github.com/circonus-labs/circonus-kubernetes-agent/pull/91/commits/d71873af18c6bbca5d8fb9c2066daf06d888b9f6)